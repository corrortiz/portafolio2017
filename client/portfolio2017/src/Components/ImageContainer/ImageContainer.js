import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import GlobalsConnect from '../../HOC/GlobalsConnect/GlobalsConnect';
import { Contratanos, AoDesarrollo } from '../../Assets/diccionary';
import { lenguajeSelector } from '../../Store/Actions/globals';

class ImageContainer extends Component {
  render() {
    const { lenguaje } = this.props.globals;
    return (
      <div className="ImageContainer">
        <h1 className="ImageContainer__main-text">
          {lenguajeSelector(lenguaje, AoDesarrollo)}
        </h1>
        <span className="ImageContainer__secondary-text">
          The sky's the limit
        </span>
        <Link to={'/contact'}>
          <a className="btn btn--white btn--animated ImageContainer__btn">
            {lenguajeSelector(lenguaje, Contratanos)}
          </a>
        </Link>
      </div>
    );
  }
}

export default GlobalsConnect(ImageContainer);
