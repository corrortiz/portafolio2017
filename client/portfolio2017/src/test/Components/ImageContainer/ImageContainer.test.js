import React from 'react';
import { shallow } from 'enzyme';
import { ImageContainer } from '../../../Components/ImageContainer/ImageContainer';

describe('ImageContainer Commponent', () => {
  it('Should revise a snapchopt of ImageContainer', () => {
    const wrapper = shallow(<ImageContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
