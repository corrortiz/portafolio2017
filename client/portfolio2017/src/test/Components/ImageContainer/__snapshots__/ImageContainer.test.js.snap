// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageContainer Commponent Should revise a snapchopt of ImageContainer 1`] = `
<ImageContainer
  classes={
    Object {
      "root": "ImageContainer-root-1",
    }
  }
/>
`;
